cypress-tests/cypress/e2e/PaymentUtils/ItauBank.js

```diff
@@ -0,0 +1,68 @@
+export const connectorDetails = {
+    card_pm: {
+        ZeroAuthMandate: {
+            Response: {
+              status: 500,
```

Comment:
Is this intentional? 4xx is expected here if I'm not wrong. Also, this can be picked from defaults which is in `commons.js`